src/Controller/GameController.tsx

```diff
@@ -1,11 +1,15 @@
-import { LEVEL_UP_ROW_COUNT, MAX_LEVEL } from '../utils/Constants';
+import { LEVEL_UP_ROW_COUNT, MAX_LEVEL, POINTS_PER_LINE } from '../utils/Constants';
 
 export type GameStatus = 'initial' | 'pause' | 'playing' | 'gameOver';
 
 export const calculateScore = (currentScore: number, points: number) => {
   return currentScore + points;
 };
 
+export const calculatePoints = (rowsDeleted: number): number => {
+  return POINTS_PER_LINE.get(rowsDeleted);
+};
```

Comment:
Using `Map` is overhead

```suggestion
export const calculatePoints = (rowsDeleted: number): number => {
  return [0, 100, 300, 700, 1500][rowsDeleted];
};
```

or use the regularity of the series:

```suggestion
export const calculatePoints = (rowsDeleted: number): number => {
  if (rowsDeleted <= 0) return 0;
  return calculatePoints(rowsDeleted - 1) * 2 + 100;
};
```

but recursion is also overhead in this case